Installation.md

```diff
@@ -4,48 +4,6 @@ The following are instructions to set up a production server for MarkUs.
 
 The following steps are for installation on a machine running Ubuntu 20.04 and all examples given below assume that you are installing in such an environment. Some changes may be required if installing on other operating systems.
 
-## System Requirements
-
-Ensure the following ubuntu packages are installed:
-
-- build-essential : (needed to install node/npm)
```

Comment:
@mishaschwartz I tested out https://github.com/MarkUsProject/Markus/pull/6246 and the changes all look good. 👍

One thing though: I really want us to keep track of the *purpose* of various system dependencies somewhere. If the `.control` file syntax support comments can we add some to that file?

Reply:
updated in the PR on Markus